findbugs/build.gradle

```diff
@@ -286,19 +204,6 @@ publishing {
         classifier 'javadoc'
       }
     }
-
-    annotations(MavenPublication) {
```

Comment:
Yes we need. I'm working on another branch `issue-30`.

https://github.com/KengoTODA/spotbugs/commits/issue-30